app/models/lawful_presence_determination.rb

```diff
@@ -84,6 +84,7 @@ def start_ssa_process
 
       if result.failure? && EnrollRegistry.feature_enabled?(:validate_and_record_publish_errors)
         process_ssa_request_failure(result, ssa_verification_type)
+        ::Operations::Eligibilities::BuildFamilyDetermination.new.call(family: self.ivl_role.person.primary_family, effective_date: TimeKeeper.date_of_record)
```

Comment:
This request can fail for multiple reason. But as per current ticket requirement if the failure is for **no VLP documents** error then reset the due date.